scripts/calcInterest.py

```diff
@@ -0,0 +1,26 @@
+# Bailee Segars
+import pandas as pd
+from decimal import *
```

Comment:
```suggestion
from decimal import Decimal
```
import only what we need from module